### ISSUE TITLE
DetailsList: sometimes isMultiline style doesn't work accidentally

### ISSUE THREAD

**Issue by nekoya** [OfficeDev/office-ui-fabric-react#5814]
Sometimes the content in a cell displays on single line and some of them are hidden in spite of enable `isMultiline`.

expected

<img width="326" alt="2018-08-06 14 50 55" src="https://user-images.githubusercontent.com/74617/43713258-e78b8796-99b3-11e8-8429-f79b03f99dbe.png">

in case of hidden content.

<img width="323" alt="2018-08-06 14 51 08" src="https://user-images.githubusercontent.com/74617/43713257-e7689240-99b3-11e8-82a6-d8d4095bdde7.png">

I didn't encounter the situation like that before v6.27.0. I suspect that CSS priorities are not ideal in some cases.

Nowadays, DetailsList is implemented by CSSinJS, placed `packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.styles.ts`.

Besides, in the former version (v6.27.0), these styles are declared at `DetailsRow.scss`.

Classes are declared at `DetailsRow.styles.ts` as JavaScript object form, order of the properties are not declarable. Is it affects these cases?

HTML elements has suffixed class names like 'cell-175' in actual, thus my prediction may misunderstanding.

Does anyone encountering this stuation?

In addition, I met circumstances like this when I replace the content dynamically through XHR. Class name like `cell-XXX` changed dynamically too.

I found the workaround, remove `whiteSpace: 'nowrap'` from `defaultCellStyles`, then the contents shows as expected. However this approach is not ideal of course.

**Comment by kenotron** [comment]
I really cannot repro this. I've modified an example here locally to dynamically force update the details list item contents in a multiline column. I simply cannot see this case where the content is being clipped. 

The css() call in DetailsRowFields is probably at play here, but it would have ordered the cell class names in a predetermined order. I'm closing this for now, please re-open with a minimal repro.

**Comment by kanufy** [comment]
Your web site occurs same problem, too.

![officceui](https://user-images.githubusercontent.com/8322617/44079552-57bcfb86-9fe4-11e8-95a1-10035038759c.gif)

It seems `isMultiline-XXX` CSS is overwritten.

**Comment by kenotron** [comment]
So, @Jahnp, can you publish a new update to the website? I don't see this repro locally at all. The fabric-website AND demo sites both do not exhibit this behavior (can you make sure the mergestyles version is updated as well?)

**Comment by KevinTCoughlin** [comment]
For linking this should have been fixed w/ https://github.com/OfficeDev/office-ui-fabric-react/pull/5930/files#diff-39279b6555df6a42c15227ace56e979dR319

**Comment by Jahnp** [comment]
Yes--I've been trying to get the site updated the last few days, but there've been issues deploying due to a stalled agent queue. That seems to be resolved now, and I've got another update queued up. I should have an update on this in a few hours.

**Comment by kanufy** [comment]
In firefox(61.0.1), I face to this problem, too.

![office-firefox](https://user-images.githubusercontent.com/8322617/44327393-797d3380-a499-11e8-8947-811632abaf23.gif)

my operations are

- Open `implementation` in fabric website docs ( because used DetailsList)
  - ex. https://developer.microsoft.com/en-us/fabric#/components/detailslist#Implementation
- Click rows having long content in multiline
sample↓
![image](https://user-images.githubusercontent.com/8322617/44327651-34a5cc80-a49a-11e8-8b6d-a59b589690fe.png)

if I do this action, rows does not display multiline.
In short, rows display only a part.

### Concrete Troubles
- I can't copy the row having multiline long content
- I feel visual confusion

**Comment by micahgodbolt** [comment]
The updates to the cell styles are overriding the isMultiline styles

![image](https://user-images.githubusercontent.com/1434956/44865378-057c2180-ac37-11e8-957d-b304274f3839.png)

specifically the white-space value.

**Comment by micahgodbolt** [comment]
Fixed by #5930
